deck_utils.py

```diff
@@ -0,0 +1,37 @@
+import asyncio
+import random
+from typing import List
+from temporalio import activity
+from card import Card, Suit, Rank
+from game_state import GameState
+
+
+def create_deck() -> List[Card]:
+    return [Card(suit, rank) for suit in Suit for rank in Rank]
+
+
+@activity.defn
+async def shuffle_deck(deck: List[Card], seed: int) -> List[Card]:
+    await asyncio.sleep(0.1)
+    return await shuffle_deck_sync(deck, seed)
```

Comment:
U: Why is there a `shuffle_deck` and a `shuffle_deck_sync` separate?

Reply:
It's unnecessary; don't know what I was thinking. I've fixed this.